front/App.tsx

```diff
@@ -1,15 +1,28 @@
 import { NativeBaseProvider } from "native-base";
 import Theme from './Theme';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { Provider } from 'react-redux';
 import store from './state/Store';
 import { Router } from './Navigation';
 import './i18n/i18n';
+import * as SplashScreen from 'expo-splash-screen';
 
 const queryClient = new QueryClient();
 
 export default function App() {
+
+	SplashScreen.preventAutoHideAsync();
+	setTimeout(SplashScreen.hideAsync, 2000);
```

Comment:
Is this the timeout for the splash screen, like I have to watch the splash screen for 2 seconds to access the app, If yes I think 500ms should be better